### ISSUE TITLE
[Add/Edit Observation] Forms containing a photo field appear blank

### ISSUE THREAD

**Issue by gino-m** [google/ground-platform#651]
**To Reproduce**
1. Create a layer + a form with a photo field
2. Add a point to that layer
2. Add/edit an observation

**Expected behavior**
The edit observation form is displayed

**Actual Behavior**
A blank for appears, with this error logged to the console:

```
main-es2018.cd8e2ed269b8a0c84033.js:1 ERROR Error: Unimplemented conversion to FormControl(s) for Field with
             Type:3
    at z.convertObservationToFormGroup (main-es2018.cd8e2ed269b8a0c84033.js:1)
    at z.initForm (main-es2018.cd8e2ed269b8a0c84033.js:1)
    at z.onSelectObservation (main-es2018.cd8e2ed269b8a0c84033.js:1)
    at u._next (main-es2018.cd8e2ed269b8a0c84033.js:1)
    at u.__tryOrUnsub (main-es2018.cd8e2ed269b8a0c84033.js:1)
    at u.next (main-es2018.cd8e2ed269b8a0c84033.js:1)
    at l._next (main-es2018.cd8e2ed269b8a0c84033.js:1)
    at l.next (main-es2018.cd8e2ed269b8a0c84033.js:1)
    at o.next (main-es2018.cd8e2ed269b8a0c84033.js:1)
    at o.next (main-es2018.cd8e2ed269b8a0c84033.js:1)
```

**Screenshots**

![image](https://user-images.githubusercontent.com/228050/116754458-c352a180-a9d6-11eb-81b2-3753e5d94633.png)

@gyarill Could you PTAL? The form should log and ignore unsupported fields rather than throwing an error.

**Comment by gino-m** [comment]
For example in dev db: /project/0GBtE5FZuxtdTLMyYfuO#f=6Auxm8X5VckLKa16hNWk

**Comment by gino-m** [comment]
@DaoyuT I think this likely will be solved by #640. @gyarill hasn't been available; would you like to pull that PR to address comments and merge?
